src/GetAPI.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace TheRealMVP;
+
+class GetAPI
+{
+    private array $json;
+    public function __construct()
+    {
+        $curlConnection = curl_init();
+        curl_setopt($curlConnection, CURLOPT_URL, 'https://dev.maydenacademy.co.uk/resources/sports_teams/sports.json');
+        curl_setopt($curlConnection, CURLOPT_RETURNTRANSFER, true);
+        $apiData = curl_exec($curlConnection);
+        curl_close($curlConnection);
+        $this->json = json_decode($apiData, true);
```

Comment:
I would suggest refactoring this slightly. 

It makes sense that the curl connection is created upon instantiation, but I don't love the way that the request is also sent and the connection is closed in one go. In its current state, adding additional API requests would require rewriting the whole class.

What about if your constructor just created the connection and stored it on a property, and then your getJson method actually executed the request and returned the JSON data using the connection property.

Then, you could make use of a destruct method to close the curl connection once this object is finished with.

This way, adding new requests (if the API was expanded) would just be a case of adding another getJson type method but for a different URL.